### PR TITLE
Noise in LAr calorimeter

### PR DESCRIPTION
**Pull request by zaborowska**
Changes proposed in this pull-request:

- fix bug in tool header
- zero segmentation cellID so volumes can be found properly when creating a map of all existing cells
- add new test that runs reconstruction using noise values from file (for ecal barrel)

ENDCAP:
- add envelopes for the layers to ensure proper position retrieval
- add tests checking the positions
- extend utilites to allow estimation of number of cells in the TGeoCone

- new algorithm useful for rewrtting the bitfield (e.g. change of order, removal of some field)

To be done:

- [x] move /afs/cern.ch/user/n/novaj/public/elecNoise_sfcorrection_50Ohm_default_differentTraces.root to our eos space

### CODE REVIEW

File: Reconstruction/RecCalorimeter/tests/options/runFullCaloSystem_ReconstructionSW_noiseFromFile.py [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+from Configurables import AuditorSvc, ChronoAuditor
+chra = ChronoAuditor()
+audsvc = AuditorSvc()
+audsvc.Auditors = [chra]
+podioinput.AuditExecute = True
+createClusters.AuditExecute = True
+out.AuditExecute = True
+
+ApplicationMgr(
+    TopAlg = [podioinput,
```
Comment (faltovaj) [inline Reconstruction/RecCalorimeter/tests/options/runFullCaloSystem_ReconstructionSW_noiseFromFile.py]: I think 'createEcalBarrelCells' should be called here.

**Comment by zaborowska** [comment]
retest this please

**Comment by zaborowska** [comment]
retest this please

**Comment by zaborowska** [comment]
retest this please

**Pull request merged**